cropper/app/model/ExportRequest.scala

```diff
@@ -0,0 +1,45 @@
+package model
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+
+import com.gu.mediaservice.model._
+
+
+sealed trait ExportRequest {
+  val uri: String
+}
+
+case class FullExportRequest(uri: String) extends ExportRequest
+
+case class CropRequest(uri: String, bounds: Bounds, aspectRatio: Option[String]) extends ExportRequest
+
+
+object ExportRequest {
+
+  private val aspectRatioLike = """\d+:\d+""".r
+
+  private val readCropRequest: Reads[CropRequest] = (
+    (__ \ "source").read[String] ~
+    __.read[Bounds] ~
+    (__ \ "aspectRatio").readNullable[String](pattern(aspectRatioLike))
+  )(CropRequest.apply _)
+
+  private val readFullExportRequest: Reads[FullExportRequest] =
+    (__ \ "source").read[String].map(FullExportRequest.apply)
+
+  val readExportRequest: Reads[ExportRequest] = Reads[ExportRequest]( jsvalue => (jsvalue \ "type") match {
```

Comment:
if `(jsvalue \ "type")` isn't there, what does it return? Or does it error if we haven't told it to return an `Option`?